src/pybind/mgr/dashboard/services/access_control.py

```diff
@@ -25,7 +25,7 @@
 from ..settings import Settings
 
 logger = logging.getLogger('access_control')
-
+FileName = 'password/secret'
```

Comment:
This looks like a constant, so what about following [pep8](https://www.python.org/dev/peps/pep-0008/#constants) and rename it i.e. ```DEFAULT_FILE_CONTENT```?

Reply:
Done